addons/lunch/static/src/components/lunch_dashboard.js

```diff
@@ -63,10 +63,7 @@ export class LunchOrderLine extends Component {
 
 export class LunchAlert extends Component {
     static props = ["message"];
-    static template = xml`<t t-out="message"/>`;
-    get message() {
-        return markup(this.props.message);
```

Comment:
Note: strong anti-pattern to blindly markup props, markup should be when we receive them from the server and know they are safe.